H2TauTau/scripts/multiHarvest.py

```diff
@@ -53,63 +50,46 @@ def multithreadmap(f,X,ncores=20, **kwargs):
     p.terminate()
     return(Xout)
 
+def find_dirs_in_dir(basepath, to_match=''):
+    if to_match == '':
```

Comment:
This condition is not necessary if you set `to_match` default value to `"''"`, as is the default value for `options.cut_on_sample_names` as `echo random | grep ''` returns `random`.

Reply:
great idea thanks!